### PR TITLE
Feature/cards

### LINKED ISSUE #7: Component | Available Craft Container
- [ ] structure, plan and style container component for all available crafts

### LINKED ISSUE #8: Component | Craft Card
- [ ] structure, plan and style craft cards

### PR DESCRIPTION
**Pull request by Abumpus1**
## What was changed:
- Implemented ItemCards in the card container
- passed down mock data to item cards from card container
- added more mock data

## Explain the reason for changes: (If there is no spec file how were things tested)
- To better match wireframe

## Link to issue: (Delete if no issue)
closes #7 
closes #8

### CODE REVIEW

**Review by Wperricone — COMMENTED** [review]
Looks great especially like the styled components

File: src/Components/ItemCard.js [hunk 1]
```diff
@@ -1,20 +1,58 @@
...
-    )
+        <ItemCardSection className="item-card">
+           <ItemCardImg className="stock-img" src={require("../assets/myrlene-numa-SnITZTTeJVE-unsplash.jpg")} alt="craft-item" />
+           <ItemBoxDiv>
+                <ItemH4>{name}</ItemH4>
+                <ItemP>Amount: {amount}</ItemP>
+                <ItemP>Crafty Joe</ItemP>
+           </ItemBoxDiv>
+        </ItemCardSection>
+    );
```
Comment (JoeRupp) [inline src/Components/ItemCard.js:14]: Nice simple component 👍

**Review by stephanie-roe — APPROVED** [review]
Looking awesome!

**Comment by JoeRupp** [comment]
Great work!

**Pull request merged**